src/UsgsAstroLsSensorModel.cpp

```diff
@@ -2766,7 +2773,7 @@ std::string UsgsAstroLsSensorModel::constructStateFromIsd(const std::string imag
 
 
    state["m_parameterVals"] = std::vector<double>(NUM_PARAMETERS, 0.0);
-   state["m_parameterVals"][15] = state["m_focal"].get<double>();
+   // state["m_parameterVals"][15] = state["m_focal"].get<double>();

```

Comment:
We are storing focal length in two places. We should either only store it in one place or only use on in calculations, see line 1733.

Reply:
I think that we want it in the parameterVals? If we want to do an adjustment and let the focal length vary, we need to be able to make modifications to the adjustments parameter. I think the only way to vary focal is if it is in the parameterVals vector.

Reply:
There's 2 things we should do then.

1. Remove explicit focal length stuff from our metadata stuff.
2. Remove m_focal from all calculations and use adjusted parameter 15.

I will do 2 on this PR.